spec/es_spec_helper.rb

```diff
@@ -34,14 +34,26 @@ def get_client
 RSpec.configure do |config|
   config.include ESHelper
 
-  config.before(:suite, :integration => true) do
-    @docker = Longshoreman.new("#{CONTAINER_IMAGE}:#{CONTAINER_TAG}", CONTAINER_NAME)
-    # TODO(talevy): detect when ES is up and ready instead of sleeping
-    # an arbitrary amount
-    sleep 10
+  config.before(:all, :integration => true) do
```

Comment:
Can we add a comment here why we use `:all` vs `:suite`? For the sake of the next person looking at it?